.github/actions/build-zui/action.yml

```diff
@@ -47,16 +51,45 @@ runs:
         security find-identity -p codesigning -v
       shell: bash
 
+    - name: Checkout esigner-codesign repository
+      if: runner.os == 'Windows'
+      uses: actions/checkout@v3
+      with:
+       repository: 'SSLcom/esigner-codesign'
+       path: esigner-codesign
+
+    - name: Make values from package.json available in Actions steps
+      id: zui-package
+      uses: RadovanPelka/github-action-json@v1.0.1
+      with:
+        path: apps/zui/package.json
+
     - name: Build & Publish
       run: ${{ inputs.cmd }}
       shell: bash
       env:
         GH_TOKEN: ${{ inputs.gh_token }}
-        WIN_CSC_KEY_PASSWORD: ${{ inputs.csc_key_password }}
-        WIN_CSC_LINK: ${{ inputs.csc_link }}
         APPLE_ID: ${{ inputs.apple_id }}
         APPLE_ID_PASSWORD: ${{ inputs.apple_id_password }}
         APPLE_TEAM_ID: ${{ inputs.apple_team_id }}
+        CODE_SIGN_SCRIPT_PATH: ${{ github.workspace }}/esigner-codesign/dist/index.js
+        INPUT_COMMAND: sign
+        INPUT_FILE_PATH: ${{ github.workspace }}/dist/apps/zui/${{ fromJSON(steps.zui-package.outputs.productName) }} Setup ${{ fromJSON(steps.zui-package.outputs.version) }}.exe
```

Comment:
The `fromJSON()` wrappers are needed because the Action I added above to pull values from the `package.json` delivers strings with quotes around them, and when these remained intact the esigner tool interpreted them as parts of the filename and choked.